spring-integration-cassandra/build.gradle

```diff
@@ -36,11 +36,11 @@ sourceCompatibility = targetCompatibility = 1.7
 
 ext {
 	cassandraVersion = '2.1.5'
-	cassandraUnitVersion = '2.1.3.1'
+	cassandraUnitVersion = '2.1.9.2'
 	jacocoVersion = '0.7.2.201409121644'
 	slf4jVersion = '1.7.12'
-	springDataCassandraVersion = '1.3.0.RELEASE'
-	springIntegrationVersion = '4.2.0.RELEASE'
+	springDataCassandraVersion = '1.3.2.RELEASE'
+	springIntegrationVersion = '4.2.3.RELEASE'
```

Comment:
Well, I'm reaching successful build with this configuration:
```
	cassandraUnitVersion = '2.2.2.1'
	jacocoVersion = '0.7.2.201409121644'
	slf4jVersion = '1.7.13'
	springDataCassandraVersion = '1.3.2.RELEASE'
	springIntegrationVersion = '4.2.4.RELEASE'
```
And 
```
dependencies {
	compile "org.springframework.integration:spring-integration-core:$springIntegrationVersion"
    compile ("org.springframework.data:spring-data-cassandra:$springDataCassandraVersion") {
		exclude group: 'org.liquibase', module: 'liquibase-core'
	}

	testCompile "org.springframework.integration:spring-integration-test:$springIntegrationVersion"

    testCompile("org.cassandraunit:cassandra-unit-spring:$cassandraUnitVersion")

	testRuntime "org.slf4j:slf4j-log4j12:$slf4jVersion"

	jacoco "org.jacoco:org.jacoco.agent:$jacocoVersion:runtime"
}
```
below.

Yes, there is something like:
```
10:58:06.570 ERROR [OptionalTasks:1][org.apache.cassandra.service.CassandraDaemon] Exception in thread Thread[OptionalTasks:1,5,main]
java.lang.NoSuchMethodError: com.codahale.metrics.Snapshot: method <init>()V not found
	at com.codahale.metrics.UniformSnapshot.<init>(UniformSnapshot.java:39)
	at org.apache.cassandra.metrics.EstimatedHistogramReservoir$HistogramSnapshot.<init>(EstimatedHistogramReservoir.java:77)
```
in the logs for `CassandraMessageHandlerTests`, but it is different story and we can go ahead with this and consider the proper `@EmbeddedCassandra` in the next JIRA.

Let's fix this PR particularly for the Namespace support!